### ISSUE TITLE
Question at loss.py

### ISSUE THREAD

**Issue by Seyoung9304** [YeLyuUT/SSeg#6]
Hello! Thank you for the great work. 

I have a question at [loss.py - ImgWtLossSoftNLL - forward()](https://github.com/YeLyuUT/SSeg/blob/94107a724d85f991590aa2da94b2090e80a6998f/loss.py#LL188C53-L188C53). 

As far as I understand, from **line 182 to 188**, we are calculating loss for each image, iterating as much as the batch size. 
However, when calling the `custom_nll` function, `weights` does not give in batch units, but delivers the whole as a factor. 
I think it's right to fix it with `weights[i]`, can you check if I'm right?

```
# As-is
for i in range(0, inputs.shape[0]):
            if not self.batch_weights:
                class_weights = self.calculate_weights(target_cpu[i])
            loss = loss + self.custom_nll(inputs[i].unsqueeze(0),
                                          target[i].unsqueeze(0),
                                          class_weights=torch.Tensor(class_weights).cuda(),
                                          border_weights=weights, mask=ignore_mask[i])
```

```
# To-be
for i in range(0, inputs.shape[0]):
            if not self.batch_weights:
                class_weights = self.calculate_weights(target_cpu[i])
            loss = loss + self.custom_nll(inputs[i].unsqueeze(0),
                                          target[i].unsqueeze(0),
                                          class_weights=torch.Tensor(class_weights).cuda(),
                                          border_weights=weights[i], mask=ignore_mask[i])
```

Thank you in advance. :)
